servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -174,6 +174,39 @@ def health():
     return HealthResponse(available=True)
 
 
+def get_shard_attempts(id, task, index, **kwargs):
+    """
+    Query for attempt metadata for a specified job task shard
+
+    :param id: Job ID
+    :type id: str
+
+    :param task: Task Name
+    :type task: str
+
+    :param index: Shard Index
+    :type index: str
+
+    :rtype: JobAttemptsResponse - never actually returned
+    """
+    raise NotImplemented('Scattered jobs not supported by dsub.')
```

Comment:
I guess you wanted `NotImplementedError('blah')` here, check: https://help.semmle.com/wiki/display/PYTHON/NotImplemented+is+not+an+Exception

Reply:
fixed